relay_test.go

```diff
@@ -89,17 +89,23 @@ func TestRelay(t *testing.T) {
 	})
 }
 
-func DisabledTestRelayHandlesCrashedPeers(t *testing.T) {
-	withRelayedEcho(t, func(_, server, client *Channel, ts *testutils.TestServer) {
-		ctx, cancel := NewContext(time.Second)
+func TestRelayHandlesClosedPeers(t *testing.T) {
+	opts := serviceNameOpts("test").SetRelayOnly().
```

Comment:
Chaining is pretty uncommon in Go generally, I tend to only use it in tests for options. Generally it's all on one line or each option on a separate line, but in this set of tests, the `SetRelayOnly` is always required, and all the tests have the same pattern of `serviceNameOpts("..").SetRelayOnly`, so I left it one line to match the rest of the tests.